openmls/src/group/core_group/test_core_group.rs

```diff
@@ -70,122 +68,112 @@ pub fn flip_last_byte(ctxt: &mut HpkeCiphertext) {
 
 #[apply(ciphersuites_and_backends)]
 fn test_failed_groupinfo_decryption(
-    ciphersuite: &'static Ciphersuite,
+    ciphersuite: Ciphersuite,
     backend: &impl OpenMlsCryptoProvider,
 ) {
-    for version in Config::supported_versions() {
```

Comment:
This would then be something we'd do for all elements of the `ProtocolVersions` enum, right? Although we might want to do that with a macro in the future.

Reply:
Yes, I made it so that the code actually only supports MLS1.0 because this was the only place we actually support something else. When we start adding support for other versions we should indeed again test all of them, but with the macro and everywhere.